pkg/seedadmissioncontroller/webhooks/admission/extensionresources/admission.go

```diff
@@ -16,266 +16,290 @@ package extensionresources
-				return validation.ValidateDNSRecordUpdate(n.(*extensionsv1alpha1.DNSRecord), o.(*extensionsv1alpha1.DNSRecord))
-			},
-		},
-
-		gvrDruid("etcds"): {
-			newObject: func() client.Object { return new(druidv1alpha1.Etcd) },
-			validateCreateResource: func(n, _ client.Object) field.ErrorList {
-				return druidvalidation.ValidateEtcd(n.(*druidv1alpha1.Etcd))
-			},
-			validateUpdateResource: func(n, o client.Object) field.ErrorList {
-				return druidvalidation.ValidateEtcdUpdate(n.(*druidv1alpha1.Etcd), o.(*druidv1alpha1.Etcd))
-			},
-		},
-
-		gvr("extensions"): {
-			newObject: func() client.Object { return new(extensionsv1alpha1.Extension) },
-			validateCreateResource: func(n, _ client.Object) field.ErrorList {
-				return validation.ValidateExtension(n.(*extensionsv1alpha1.Extension))
-			},
-			validateUpdateResource: func(n, o client.Object) field.ErrorList {
-				return validation.ValidateExtensionUpdate(n.(*extensionsv1alpha1.Extension), o.(*extensionsv1alpha1.Extension))
-			},
-		},
-
-		gvr("infrastructures"): {
-			newObject: func() client.Object { return new(extensionsv1alpha1.Infrastructure) },
-			validateCreateResource: func(n, _ client.Object) field.ErrorList {
-				return validation.ValidateInfrastructure(n.(*extensionsv1alpha1.Infrastructure))
-			},
-			validateUpdateResource: func(n, o client.Object) field.ErrorList {
-				return validation.ValidateInfrastructureUpdate(n.(*extensionsv1alpha1.Infrastructure), o.(*extensionsv1alpha1.Infrastructure))
-			},
-		},
-
-		gvr("networks"): {
-			newObject: func() client.Object { return new(extensionsv1alpha1.Network) },
-			validateCreateResource: func(n, _ client.Object) field.ErrorList {
-				return validation.ValidateNetwork(n.(*extensionsv1alpha1.Network))
-			},
-			validateUpdateResource: func(n, o client.Object) field.ErrorList {
-				return validation.ValidateNetworkUpdate(n.(*extensionsv1alpha1.Network), o.(*extensionsv1alpha1.Network))
-			},
-		},
-
-		gvr("operatingsystemconfigs"): {
-			newObject: func() client.Object { return new(extensionsv1alpha1.OperatingSystemConfig) },
-			validateCreateResource: func(n, _ client.Object) field.ErrorList {
-				return validation.ValidateOperatingSystemConfig(n.(*extensionsv1alpha1.OperatingSystemConfig))
-			},
-			validateUpdateResource: func(n, o client.Object) field.ErrorList {
-				return validation.ValidateOperatingSystemConfigUpdate(n.(*extensionsv1alpha1.OperatingSystemConfig), o.(*extensionsv1alpha1.OperatingSystemConfig))
-			},
-		},
-
-		gvr("workers"): {
-			newObject: func() client.Object { return new(extensionsv1alpha1.Worker) },
-			validateCreateResource: func(n, _ client.Object) field.ErrorList {
-				return validation.ValidateWorker(n.(*extensionsv1alpha1.Worker))
-			},
-			validateUpdateResource: func(n, o client.Object) field.ErrorList {
-				return validation.ValidateWorkerUpdate(n.(*extensionsv1alpha1.Worker), o.(*extensionsv1alpha1.Worker))
-			},
-		},
+func AddWebhooks(mgr manager.Manager) error {
```

Comment:
Doc strings for many exported functions in this file are missing

Reply:
This is the case for old commit it has been updated in this one c3569e03fa2d7113eae24dce5ff952dfcbc37dcc